specification/terraform/Microsoft.AzureTerraform.Management/routes.tsp

```diff
@@ -2,52 +2,67 @@ import "@azure-tools/typespec-azure-core";
 import "@typespec/rest";
 import "./models.tsp";
 import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
 
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using Azure.Core;
+using Azure.ClientGenerator.Core;
 using Azure.ResourceManager;
 using Azure.ResourceManager.Foundations;
 using OpenAPI;
 
 namespace Microsoft.AzureTerraform;
 
-#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "Cannot use @armResourceOperations decorator here, the auto-generated routes do not match feature requirements"
-#suppress "@azure-tools/typespec-azure-core/no-openapi" "TODO: migrate to LRO concepts DO NOT USE x-ms-long-running-operation-options"
-@doc("Exports the Terraform configuration of the specified resource(s).")
-@route("/subscriptions/{subscriptionId}/providers/Microsoft.AzureTerraform/exportTerraform")
-@post
-@tag("ExportTerraform")
-@extension(
-  "x-ms-long-running-operation-options",
-  {
-    `final-state-via`: "azure-async-operation",
-    `final-state-schema`: "#/definitions/OperationStatus",
-  }
-)
-op exportTerraform(
-  ...ApiVersionParameter,
-  ...SubscriptionIdParameter,
+@doc("Parameters common to all ExportTerraform operations")
+model CommonExportTerraformParameters {
+  ...ApiVersionParameter;
+  ...SubscriptionIdParameter;
+}
```

Comment:
looks like this is not used anywhere, maybe we could remove this or utilize it?

Reply:
Ah thx, this is an oversight. Fixed in e669bc3b30.